### PR TITLE
Add UnFUCK Event article

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Ready for first review and suggestions here.

### CODE REVIEW

File: content/blog/2016/Unfuck-2016/article.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - programming
+  - event
+  - unfuck
+  - alumini
+  - conference
+---
+
+UnFUCK 2016! It is happening.
+
+UnFUCK is a event organized and held by the UnFUG group. It features a lot of
```
Comment (TheNeikos) [inline content/blog/2016/Unfuck-2016/article.md]: 'several' instead of 'a lot of'

File: content/blog/2016/Unfuck-2016/article.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+talks with topics varying from kernel foo to cloud deployment, hacker games, a
```
Comment (TheNeikos) [inline content/blog/2016/Unfuck-2016/article.md]: How about 'dev' instead of 'foo'

**Comment by matthiasbeyer** [comment]
@TheNeikos fixed.

**Comment by TheNeikos** [comment]
The switch from future to present seems a bit weird still, but it should work.

**Pull request merged**